lib/data_script.rb

```diff
@@ -0,0 +1,9 @@
+class DataScript
```

Comment:
This will be a base class for these sorts of data scripts going forward.

The idea here is to 
* default to a dry run mode to see what would be changed before running things
* provide a standard scaffold for things like logging and capturing results

Reply:
I like the idea 👍 

Think this is worth documenting under `/doc/arch`, maybe as a follow up?
Our current data migration strategy is outlined at [007-robust-data-migrations.md](https://github.com/simpledotorg/simple-server/blob/master/doc/arch/007-robust-data-migrations.md )